PKHeX.WinForms/Program.cs

```diff
@@ -93,6 +94,12 @@ private static void CurrentDomain_UnhandledException(object sender, UnhandledExc
                 // Todo: make this translatable
                 ErrorWindow.ShowErrorDialog("An unhandled exception has occurred.\nPKHeX must now close.", ex, false);
             }
+            catch (MissingMethodException)
+            {
+                // Todo: make this translatable
+                MessageBox.Show(".NET Framework 4.6 needs to be installed for this version of PKHeX to run.", "PKHeX Error", MessageBoxButtons.OK, MessageBoxIcon.Stop);
+                Process.Start(@"https://www.microsoft.com/en-us/download/details.aspx?id=48130");
```

Comment:
Spun a VM. You're right, it doesn't work.

.NET Framework 3.5 (Vanilla Windows 7)
![pkhex-win7sp1_3 5](https://cloud.githubusercontent.com/assets/2667537/22495319/3ccfde4c-e87a-11e6-9af9-26154ca7aa8a.png)

.NET Framework 4.0
![pkhex-win7sp1_4 0](https://cloud.githubusercontent.com/assets/2667537/22495324/4fc90b90-e87a-11e6-980d-a4a2594af174.png)

.NET Framework 4.5
![pkhex-win7sp1_4 5](https://cloud.githubusercontent.com/assets/2667537/22495326/54aae3fe-e87a-11e6-917c-360c0b75b3b1.png)

Perhaps we should return to .NET 4.0? I'm able to use ILMerge with *.NET Framework v4 Client* on my Win 7 VM which would support all the OSes again.